sql-gremlin/src/test/java/org/twilmes/sql/gremlin/adapter/GremlinSqlNotSupportedTest.java

```diff
@@ -49,9 +48,9 @@ public void testSubQuery() throws SQLException {
                 SqlGremlinError.UNKNOWN_OPERATOR, "SCALAR_QUERY");
     }
 
-    // TODO: Handle CAST
-    @Disabled
+    @Test
     public void testCast() throws SQLException {
-        runNotSupportedQueryTestThrows("SELECT CAST(17 AS varchar)", null);
+        runQueryTestThrows("SELECT CAST(17 AS varchar)", SqlGremlinError.UNSUPPORTED_LITERAL_EXPRESSION);
+        runQueryTestThrows("SELECT CAST(person.age as CHAR) FROM person", SqlGremlinError.UNKNOWN_OPERATOR, "CAST");
```

Comment:
This test will need to be updated to account for the SqlNotSupportedError instead.